elisp/core.el

```diff
@@ -217,4 +217,6 @@
                             mal-nil))
                          (t
                           mal-nil))))))
+
+    (elisp-eval . ,(mal-fn (lambda (string) (mal-string (format "%S" (eval (read (mal-value string))))))))
     ))
```

Comment:
I went ahead and merged.

I would probably suggest returning nil for nil/(). For the other data types, you could convert as best you can, but fallback to the string representation for things that don't have any usable conversion.